src/index.js

```diff
@@ -12,6 +13,7 @@ function Index() {
             <Routes>
                 <Route path="/" element={<Login/>}/>
                 <Route path="/register" element={<Register/>}/>
+                <Route path="/login/homepage" element={<Homepage/>}/>
```

Comment:
I think a better url route to follow would be just `/homepage`, rather than `/login/hompage`.